enterprise/core-edge/src/main/java/org/neo4j/coreedge/server/edge/EnterpriseEdgeEditionModule.java

```diff
@@ -117,18 +118,14 @@ public EnterpriseEdgeEditionModule( final PlatformModule platformModule,
 
         headerInformationFactory = TransactionHeaderInformationFactory.DEFAULT;
 
-        schemaWriteGuard = new SchemaWriteGuard()
-        {
-            @Override
-            public void assertSchemaWritesAllowed() throws InvalidTransactionTypeKernelException
-            {
-            }
-        };
+        schemaWriteGuard = () -> {};
```

Comment:
There 3 implementations like that in a code base, that should be extracted and class should be introduced instead